src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamConformanceTests.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.IO.Tests;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+namespace System.Net.Security.Tests
+{
+    public sealed class SslStreamMemoryConformanceTests : ConnectedStreamConformanceTests
+    {
+        protected override bool UsableAfterCanceledReads => false;
+        protected override bool BlocksOnZeroByteReads => true;
+        protected override Type UnsupportedConcurrentExceptionType => typeof(NotSupportedException);
+
+        protected override async Task<StreamPair> CreateConnectedStreamsAsync()
```

Comment:
Why doesn't this use the Wrapped version of the conformance tests?

Reply:
It should.